src/components/shared/ImgWithFallback.tsx

```diff
@@ -4,32 +4,58 @@ import React, { FC, ImgHTMLAttributes, useState } from 'react';
 interface IFallback {
   label: string;
   backgroundColor: string;
+  textColor: string;
 }
 
 interface IImgWithFallback extends ImgHTMLAttributes<HTMLImageElement> {
   fallback: IFallback;
 }
 
 const ImgWithFallback: FC<IImgWithFallback> = (props) => {
-  const [loadError, setLoadError] = useState(false);
```

Comment:
I flipped the logic of this component a bit to start with a loading state, and to 'prove itself' that it is not in an error instead of waiting for a failure to happen and then hiding the image.

This prevents a flash of the alt text and the broken image icon.